_posts/yyyy-mm-dd-ansible-czy-da-sie-uporzadkowac-chaos.md

```diff
@@ -0,0 +1,301 @@
+---
+layout:    post
+title:     "Ansible - czy da się uporządkować chaos"
+date:      YYYY-MM-DD HH:mm:ss +0100
+published: false
+author:    rmastalerek
+image:     /assets/img/posts/yyyy-dd-mm-ansible-czy-da-sie-uporzadkowac-chaos/ansible.png
```

Comment:
ten obrazek jakiś sensowny dorzucę, jak tekst będzie ok

Reply:
To @adampij chyba pomaga zrobić? :)

Reply:
już ogarnięte :) thx